packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -113,13 +132,26 @@ export class MarketDB extends DerivedDB {
         // This is needed to make rollbacks work properly
         doc['blockNumber'] = highestSyncedBlockNumber;
         doc['market'] = marketId;
+        doc['liquidityDirty'] = false;
         documents.push(doc);
       }
     }
 
     await this.bulkUpsertDocuments(documents);
   }
 
+  async markMarketLiquidityAsDirty(marketId: string) {
```

Comment:
What I would do instead actually is just keep this in memory and store the marketIds as a Set

Reply:
👍  updating